gqueries/output_elements/output_series/table_260_land_use_solar_wind/solar_pv_buildings/land_use_solar_wind_solar_pv_buildings_electricity_output.gql

```diff
@@ -0,0 +1,6 @@
+- query = 
+    DIVIDE(
+        V(buildings_solar_pv_solar_radiation, output_of_electricity),
+        3600 * 10**3
```

Comment:
We have a constant for this which you can use: see [constants.rb](https://github.com/quintel/etengine/blob/master/app/models/gql/runtime/functions/constants.rb) on ETEngine. You can for example use MJ_PER_MWH.